content/departments/technical-success/support/process/support-agent-customer-slack-channels.md

```diff
@@ -1,32 +1,32 @@
-## How to Create and Transfer Tickets for Support Engineering
+## How to Engage with Support
 
-We currently offer Support via Slack Channels for Enterprise customers through a Support Agent integration that must be integrated into the Slack Channel as well as email at support@sourcegraph.com. The following are steps that are set to ensure we can provide a consistent support experience to our customers in accordance with our Service Level Agreements (SLAs).
+We provide support to Enterprise customers through Slack Channels and email at support@sourcegraph.com. To ensure a consistent support experience aligned with our Service Level Agreements (SLAs), please follow these best practices:
 
-There are two methods for the Support Agent integration:
+### Best practices
 
-1. Embedding a Create Request option at the bottom of the channel (preferred option)
-2. By adding a 🎟️ emoji on the message
+1. **Avoid Blind Forwarding:** Please refrain from blindly forwarding tickets to support@sourcegraph.com. If you need to forward an email into support, include the following information to help us assist you efficiently:
 
-Below are the steps required to enable either option.
+   - Customer details
+   - Link to the specific issue
+   - Detailed summary of the issue or question
+   - Consequences of not taking the desired action
+   - Any time considerations and their significance
+   - Any additional relevant information that can assist us
 
-### Embedding the Create Request option to the Channel
+2. **Create a Ticket within Zendesk:** You can also create a ticket on behalf of the customer in Zendesk. Set the customer as the requestor, select `Apply Macro` at the bottom left of the screen, and apply the `Customer Support Ticket Request` macro. Fill out the requested information to provide comprehensive context.
```

Comment:
Can we add some sort of `NOTE` saying that if they do not have a Zendesk account an `Light Agent` account can be requested via Tech Ops?

Reply:
Yeah, that makes sense. I will add that. Thanks!